misc/metrics.py

```diff
@@ -136,7 +102,7 @@ def calculate_AP(model_output, label, anchors, grid_sizes, required_IoU=0.5):
     with torch.no_grad():
         for i in range(batch_size):
             prediction_bboxes, prediction_confidences = convert_output_to_workable_data(
-                model_output[0][i], model_output[1][i], anchors, grid_sizes)
+                model_output[0][i], model_output[1][i], anchors, grid_sizes, (320, 320))
```

Comment:
why (320, 320)?

Reply:
Because that's how the model is trained ATM